fn_spamhaus_query/fn_spamhaus_query/util/info_response.py

```diff
@@ -0,0 +1,198 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+STATIC_INFO_RESPONSE = {
```

Comment:
document why you need this table and how it's used

Reply:
ok